src/cascadia/TerminalApp/ColorHelper.h

```diff
@@ -1,7 +1,6 @@
 #pragma once
-#include "pch.h"
 
-#include <winrt/windows.ui.core.h>
+#include <winrt/Windows.UI.h>

```

Comment:
jeepers I don't love this, presumably this should be in the pch for both this project and the tests, but also those tests really haven't had love in years. So, this is fine for now. (I can only hope that it _also_ being in the pch.h for TerminalAppLib means it won't _actually_ get re-parsed here)

Reply:
The `Windows.UI` header is a smaller than the `Windows.UI.Core` header so this at least makes it a tiny bit better.